util/json_tests.c

```diff
@@ -97,7 +97,7 @@ Ensure (json, gvm_json_obj_check_str_1_when_int)
 Ensure (json, gvm_json_obj_check_str_0_and_val_when_has)
 {
   cJSON *json;
-  gchar *ret;
+  gchar *ret=NULL;
```

Comment:
```suggestion
  gchar *ret = NULL;
```

please make sure to correctly format your contributions, otherwise we won't be able to approve them.